ajax/libs/senna.js/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "senna.js",
+  "filename": "globals/senna-min.js",
+  "version": "1.2.0",
+  "description": "A blazing-fast Single Page Application engine",
+  "homepage": "http://sennajs.com/",
+  "author": {
+    "email": "edu@rdo.io",
+    "name": "Eduardo Lundgren",
+    "url": "http://eduardo.io"
+  },
+  "keywords": [
+    "spa",
+    "single",
+    "page",
+    "apps"
+  ],
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eduardolundgren/senna.js.git"
+  },
+  "npmName": "senna",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "globals/+(senna|senna-min).js",
```

Comment:
@maruilian11 but I don't attach any .map file in "globals/" folder.

> As we host "senna.js" and "senna.min.js", but not those source files which build "senna.js".
I think we don't need senna.js.map.